src/tests/test_discover.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# discover test
+python src/obs_inv_utils/obs_inv_cli.py get-obs-inventory -c src/tests/yaml_configs/test_discover_amsua_1.yaml ; 
+python src/obs_inv_utils/obs_inv_cli.py get-obs-count-meta-sinv -c src/tests/yaml_configs/test_discover_amsua_2.yaml ;
+
+
+# aws s3 clean bucket test
+python src/obs_inv_utils/obs_inv_cli.py get-obs-inventory -c src/tests/yaml_configs/test_aws_amsua_1.yaml ; 
+python src/obs_inv_utils/obs_inv_cli.py get-obs-count-meta-sinv -c src/tests/yaml_configs/test_discover_amsua_2.yaml ;
+
+echo "test complete."
```

Comment:
this is not actually a test but a chance to run some files, it won't actually check and confirm things are working. Since we are already using pytest for other testing, it would be appropriate to write this into a python file using pytest and assert statements for confirmation that the code is working appropriately